packages/starknet-snap/src/utils/logger.test.ts

```diff
@@ -0,0 +1,78 @@
+import { logger, LogLevel } from './logger';
+
+describe('Logger', () => {
```

Comment:
here I guess we only test the log level that we actually use, this is why the following are not tested ? 

```
  ERROR = 1,
  WARN = 2,
  DEBUG = 4,
  TRACE = 5,
 ```

Reply:
lets add it back